ariel-zurita/Lesson-2/homework-1/src/Car.java

```diff
@@ -0,0 +1,132 @@
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+
+public class Car {
+    private double gas;
+    private double distance;
+    private double performance;
+    private boolean powerOn;
+
+    public Car(){
+        gas = 0;
+        distance = 0;
+        powerOn = false;
+    }
+
+    public void setPerformance(double performance){
+        this.performance = performance;
+    }
+
+    public double getPerformance(){
+        return performance;
+    }
+
+    public void setGas(double gas){
+        this.gas = gas;
+    }
+
+    public double getGas(){
+        return gas;
+    }
+
+    public void setDistance(double distance){
+        this.distance = distance;
+    }
+
+    public double getDistance(){
+        return distance;
+    }
+
+    public void setPowerOn(boolean powerOn){
+        this.powerOn = powerOn;
+    }
+
+    public boolean getPowerOn(){
+        return powerOn;
+    }
+
+    public void move(double gas, double distance, double performance){
+        double usedGas = distance / performance;
+        if(gas > usedGas){
+            double newGas = gas - usedGas;
+            setGas(newGas);
+            setPowerOn(true);
```

Comment:
distance should be incremented this.distance += distance;
As a comment inside a class it is not necessary to use getters and setters to modify class members